tools/url/practice/inventoris.md

```diff
@@ -0,0 +1,15 @@
+🛠 Раньше на одном домене несколько серверов могли обрабатывать каждый своё: кто-то электронную почту, кто-то файлы, кто-то сайт, поэтому URL имели домен третьего уровня. Для почты ссылка выглядела так:
+
+```url
+mail.doka.guide
+```
+
+А для сайта так:
+
+```url
+www.doka.guide
+```
+
+Сегодня больше не пишут `www`, поскольку сайты, как правило, имеют редирект с `www` версии на обычную. При этом старая запись отнюдь не запрещена, так что это скорее дело привычки.
```

Comment:
И еще много тонкостей связанных с SEO :)

Reply:
Новичку, который только пытается понять, как устроена ссылка, на мой взгляд, было бы интересно узнать такую подробность: что за www и нужно ли его писать?

Reply:
Я немного беспокоюсь за вот такую штуку
```However, dropping www. from your main site domain can have dire consequences because of how DNS records work.```